plugin-server/package.json

```diff
@@ -60,6 +60,7 @@
         "asn1.js": "^5.4.1",
         "aws-sdk": "^2.927.0",
         "escape-string-regexp": "^4.0.0",
+        "express": "^4.18.2",
```

Comment:
_probably nothing to worry about_ since express is nicer than writing our own server but we should check plugin server startup time as we roll this out - maybe express being more complex takes longer to start - very much looking for trouble where it may not exist though

Reply:
It's definitely negligible in comparison to all the other stuff the plugin server does but yes indeed something to check